File: de.unistuttgart.iste.sqa.mpw.hamstersimulator.java/core/src/main/java/de/unistuttgart/hamster/viewmodel/impl/GameViewModelImpl.java
package de.unistuttgart.hamster.viewmodel.impl;

import de.unistuttgart.hamster.viewmodel.ViewModelCell;
import de.unistuttgart.hamster.viewmodel.GameViewModel;
import de.unistuttgart.hamster.mpw.Size;

public class GameViewModelImpl extends GameViewModel {

	public GameViewModelImpl() {

	}

	@Override
	public ViewModelCell getCellAt(int row, int column) {
		throw new RuntimeException("not implemented");
	}

	@Override
	public void init(Size size) {
		throw new RuntimeException("not implemented");
	}

}

File: de.unistuttgart.iste.sqa.mpw.karasimulator.cpp/core/src/impl/GameViewModelImpl.cpp
#include "GameViewModelImpl.h"

#include <stdexcept>

namespace viewmodel {

GameViewModelImpl::GameViewModelImpl() {
}

std::shared_ptr<viewmodel::ViewModelCell> GameViewModelImpl::getCellAt(int row,
		int column) {
	throw std::runtime_error("not implemented");
}

void GameViewModelImpl::init(mpw::Size size) {
	throw std::runtime_error("not implemented");
}

}

